Makefile

```diff
@@ -135,11 +135,11 @@ docker-feature-push: gen-semver
 
 CONTROLLER_GEN = $(shell pwd)/bin/controller-gen
 controller-gen: ## Download controller-gen locally if necessary.
-	$(call go-get-tool,$(CONTROLLER_GEN),sigs.k8s.io/controller-tools/cmd/controller-gen@v0.7.0)
+	$(call go-get-tool,$(CONTROLLER_GEN),sigs.k8s.io/controller-tools/cmd/controller-gen,v0.7.0)
```

Comment:
Is this change intentional?

Reply:
@rensyct Yes, I added the version here as a parameter so that is can be passed to "go-get-tool" that will get the package with specific version and then install it. This was done as with go version 1.18 onwards "go get" is dedicated to only adjust dependencies in go.mod. So to install external dependencies we need to "go install".